_inc/client/state/period-toggle/actions.js

```diff
@@ -0,0 +1,13 @@
+/**
```

Comment:
Unless we're planning on using this period toggle state elsewhere, we might want to make this page-specific by using `_inc/client/state/plans/actions.js` instead.

Reply:
Great suggestion. Updated to use plans folder.